### ISSUE TITLE
Frosted doesn't know about builtin function __import__

### ISSUE THREAD

**Issue by mriehl** [timothycrosley/deprecated.frosted#45]
I'm getting several name errors like this on our codebase :

```
[WARN]  frosted: src/main/python/pybuilder/pluginloader.py:39: undefined name '__import__'
```

Essentially, [`__import__` is a built-in function](http://docs.python.org/3/library/functions.html#__import__) and frosted doesn't know about it.
I think the fix is similar to [#33](https://github.com/timothycrosley/frosted/commit/16635786b60bf4fd4c3bc41fef17f31b2bc546f3) but `__import__` is a function, not a variable, so using `BUILTIN_VARS` seems odd.

I'm willing to submit a pull request if you can point me in the right direction.

**Comment by timothycrosley** [comment]
Hi @mriehl ,

Thanks for reporting this issue! I agree BUILTIN_VARS is confusing and misleading naming. I think the fix is just to come up with a more appropriate name, rename it, and then add it to the list. I'll def enthusiastically accept a pull request that does this.

Thanks!

~Tim

**Comment by timothycrosley** [comment]
Thanks again @mriehl for reporting and @mattcaldwell for fixing. This fix is now live in release 1.4.1.

~Tim
